docs/development/releasing.rst

```diff
@@ -225,18 +225,29 @@ packages that use the full bugfix/maintenance branch approach.)
       but this should *not* be done, as it might push up some unintended tags.
 
 #. If this is a release of the current release (i.e., not an LTS supported along
-   side a more recent version), update the "stable" branch to point to the new
+   side a more recent version), update the "stable" tag to point to the new
    release::
 
-      $ git checkout stable
-      $ git reset --hard v<version>
-      $ git push upstream stable --force
-
-#. Update Readthedocs so that it builds docs for the version you just released.
-   You'll find this in the "admin" tab, with checkboxes next to each github tag.
-   Also verify that the ``stable`` Readthedocs version builds correctly for
-   the new version (it should trigger automatically once you've done the
-   previous step).
+      $ git checkout v<version>
+      $ git tag -d stable
+      $ git tag -s "stable" -m "Tagging v<version> as latest stable"
+      $ git push upstream :stable
+      $ git push upstream stable
```

Comment:
This last one needs the ``--tags`` flag

Reply:
@eteq has just removed the ``--tags`` from a few line above, I think it's safer to go without it here, too?
https://github.com/astropy/astropy/pull/5640

Reply:
Are you sure it isn't needed? Tags never get pushed for me unless I explicitly add ``--tags``. Is it a git version thing?

Reply:
I always push them without, and not aware that I have any fancy settings. But by this time, we mostly got rid of all weird tags/branches, so adding it back probably won't cause any issues.

Reply:
@eteq - what shall we do with ``--tags``? As I recall, you've removed it from the instructions mostly as the old ``stable`` tag kept haunting us. Currently the tags seems mostly OK to me so should we add it back?